webapp/src/components/admin_settings/tsdb_stats/tsdb_stats.scss

```diff
@@ -0,0 +1,31 @@
+
+.tsdbstats-table__table {
+    overflow: auto;
+    width: 100%;
+    min-height: 100px;
+    max-height: 240px;
+    padding: 5px;
+    border: gray;
+    background-color: white;
+}
+
+.tsdbstats-table {
+    .table > thead > tr > th,
+    .table > tbody > tr > th,
+    .table > tfoot > tr > th,
+    .table > thead > tr > td,
+    .table > tbody > tr > td,
+    .table > tfoot > tr > td {
```

Comment:
that's super specific... does `.sdbstats-table{ th, td{ border-top: none; }}` not work?

Reply:
I copied this section from some other CSS but changed to this and verify that this also work. Thanks :)